### DISCUSSION TITLE
Load More pagination

### DISCUSSION THREAD

**Discussion by treithamer** [wpengine/faustjs#887] [Q&A] [answered]
Any example on Load More pagination? for:
https://github.com/wpengine/faustjs/blob/canary/examples/next/getting-started/src/components/Pagination.tsx

**Comment by blakewilson** [comment, marked as answer]
@treithamer We have an example of pagination in our Portfolio Blueprint example:

https://github.com/wpengine/atlas-blueprint-portfolio/blob/main/src/pages/posts/index.js#L43

We ended up creating a hook called `useNodePagination` for pagination of all the different post types:

https://github.com/wpengine/atlas-blueprint-portfolio/blob/d31539deeec4f07938d8b903106747921020d11e/src/hooks/useNodePagination.js#L14
